turbo.json

```diff
@@ -63,6 +63,7 @@
       "outputs": []
     },
     "check-types": {
+      "dependsOn": ["^build"],
```

Comment:
Do we need to build **everything** to check types or only some specific packages? I think in future we should add a pre-commit / pre-push hook to check for the types (like we do in Saleor Dashboard) but building everything could take too much time

Reply:
It builds only dependant packages, which is required to be able to check types, yes.